standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/PersistenceManagerProvider.java

```diff
@@ -251,11 +277,14 @@ private static PersistenceManagerFactory initPMF(Configuration conf, boolean for
     if (dsp == null) {
       pmf = JDOHelper.getPersistenceManagerFactory(dsProp);
     } else {
-      try {
+      String sourceName = forCompactor ? "objectstore-compactor" : "objectstore";
```

Comment:
will the pool name be as it is here or will it be something like objectstore-compactor-pool or objectstore-pool?

Reply:
The metrics of the pool would be something like {pool_name}.pool.{metric_name}, added a test:
https://github.com/apache/hive/blob/77ea96e25d01de3546c29311eadd342318b2a06a/itests/hive-unit/src/test/java/org/apache/hadoop/hive/metastore/TestMetaStoreMetrics.java#L162-L171